pipeline/bicleaner/download_pack.py

```diff
@@ -50,39 +50,68 @@ def _compress_dir(dir_path: str, compression_cmd: str) -> str:
     return compressed_path
 
 
+def check_result(result: subprocess.CompletedProcess):
+    """Checks the return code, and outputs the stdout and stderr if it fails."""
+    if result.returncode != 0:
+        print(result.stdout)
+        print(result.stderr)
+        result.check_returncode()
+
+
 def download(src: str, trg: str, output_path: str, compression_cmd: str) -> None:
-    tmp_dir = tempfile.gettempdir()
+    tmp_dir = os.path.join(tempfile.gettempdir(), "bicleaner")
+    if not os.path.exists(tmp_dir):
```

Comment:
Was not creating the temp directory the problem?

Reply:
This just helped in debugging the issue. Locally this put it in the temp directory with everything else that was in the temp directory, so it was hard to see what was being saved there.